### PR TITLE
[WIP] Updated sub channel to 4.3 for GA

### PR DESCRIPTION
**Pull request by abrennan89**
Please do not merge until requested.

### CODE REVIEW

**Comment by abrennan89** [comment]
Previous PR had SME and peer reviews completed, but was merged by accident: https://github.com/openshift/openshift-docs/pull/20884/

**Comment by abrennan89** [comment]
@kalexand-rh could you please add a `peer review done` label for this since it was already reviewed and merged previously? Just holding off the merge the reverted change until GA next week :slightly_smiling_face:

File: modules/serverless-upgrade-sub-channel.adoc [hunk 1]
```diff
@@ -5,8 +5,8 @@
 [id="serverless-upgrade-sub-channel_{context}"]
 == Upgrading the Subscription Channel
 
-In {ServerlessProductName} versions 1.5.0 and older, the only available Subscription Update Channel was `techpreview`.
-To upgrade to the latest version, you must update the channel to `preview-4.3`.
+In {ServerlessProductName} versions 1.6.0 and older, the only available Subscription Update Channel was `preview-4.3`.
```
Comment (bbrowning) [inline modules/serverless-upgrade-sub-channel.adoc:8]: Actually, the `preview-4.3` channel was only available in version 1.6.0 and not older versions. I'd remove the "and older" from this.

**Comment by abrennan89** [comment]
Covered in #21322 - closing this PR.

**Pull request closed without merging**